interfaces/portal/src/app/pages/project-settings-fsps/components/fsp-configuration-card/fsp-configuration-card.component.ts

```diff
@@ -0,0 +1,135 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  computed,
+  inject,
+  input,
+  output,
+  viewChild,
+} from '@angular/core';
+
+import {
+  injectMutation,
+  injectQuery,
+} from '@tanstack/angular-query-experimental';
+import { MenuItem } from 'primeng/api';
+import { ButtonModule } from 'primeng/button';
+import { TableModule } from 'primeng/table';
+
+import { FSP_SETTINGS } from '@121-service/src/fsps/fsp-settings.const';
+
+import { CardWithLinkComponent } from '~/components/card-with-link/card-with-link.component';
+import { EllipsisMenuComponent } from '~/components/ellipsis-menu/ellipsis-menu.component';
+import { FormDialogComponent } from '~/components/form-dialog/form-dialog.component';
+import { FspConfigurationApiService } from '~/domains/fsp-configuration/fsp-configuration.api.service';
+import { FSP_IMAGE_URLS } from '~/domains/fsp-configuration/fsp-configuration.helper';
+import { FspConfiguration } from '~/domains/fsp-configuration/fsp-configuration.model';
+import { ProjectApiService } from '~/domains/project/project.api.service';
+import { TranslatableStringPipe } from '~/pipes/translatable-string.pipe';
+import { FspConfigurationService } from '~/services/fsp-configuration.service';
+import { ToastService } from '~/services/toast.service';
+import { TranslatableStringService } from '~/services/translatable-string.service';
+
+@Component({
+  selector: 'app-fsp-configuration-card',
+  imports: [
+    TranslatableStringPipe,
+    ButtonModule,
+    FormDialogComponent,
+    TableModule,
+    CardWithLinkComponent,
+    EllipsisMenuComponent,
+  ],
+  templateUrl: './fsp-configuration-card.component.html',
+  styles: ``,
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  providers: [ToastService],
+})
+export class FspConfigurationCardComponent {
+  readonly projectId = input.required<string>();
+  readonly configuration = input.required<FspConfiguration>();
+  readonly reconfigureFsp = output<FspConfiguration>();
+
+  fspConfigurationService = inject(FspConfigurationService);
+  fspConfigurationApiService = inject(FspConfigurationApiService);
+  projectApiService = inject(ProjectApiService);
+  translatableStringService = inject(TranslatableStringService);
+  toastService = inject(ToastService);
+
+  projectAttributes = injectQuery(
+    this.projectApiService.getProjectAttributes({
+      projectId: this.projectId,
+      includeProgramRegistrationAttributes: true,
+    }),
+  );
+
+  readonly deleteConfirmationDialog = viewChild.required<FormDialogComponent>(
+    'deleteConfigurationDialog',
+  );
+
+  readonly deleteConfigurationDialogHeader = computed(
+    () =>
+      $localize`Remove` +
+      ` "${this.translatableStringService.translate(this.configuration().label) ?? ''}"`,
```

Comment:
We could also use the `fspName()`-pattern here like in "fsp-configuration-form-dialog/fsp-configuration-form-dialog"-component...